build_scripts/build_usd.py

```diff
@@ -2160,9 +2176,9 @@ def ForceBuildDependency(self, dep):
 if which("cmake"):
     # Check cmake requirements
     if Windows():
-        # Windows build depend on boost 1.70, which is not supported before
-        # cmake version 3.14
-        cmake_required_version = (3, 14)
+        # Windows build depend on boost 1.78, which is not supported before
+        # cmake version 3.24
+        cmake_required_version = (3, 24)
```

Comment:
Requiring newest cmake while not force using 1.78 seems unreasonable. A user would have to install cmake 3.24 for all windows build, even with VS2019 or VS2017

Reply:
> A user would have to install cmake 3.24 for all windows build, even with VS2019 or VS2017

Yes, I agree that it seems unreasonable.

In this PR, boost 1.78 is forced on Windows, CMake lower than 3.24 will not find boost 1.78 correctly, so CMake 3.24 is required.

Reply:
So, in my opinion, we should not force using 1.78, only if we use python > 3.8, the cmake and boost's version requirement should be present